api/test/wfl/unit/slack_test.clj

```diff
@@ -14,3 +16,23 @@
         (slack/add-notification testing-agent (testing-slack-notification)))
       (await testing-agent)
       (is (= num-msg (count (seq @testing-agent)))))))
+
+(deftest test-notify-watchers-behind-feature-switch
```

Comment:
> test-notify-watchers-behind-feature-switch tests that Slack notes are disabled unless the EV's value is "enabled"?

Yes.  But you asking this tells me I can be more clear about the test's intention :)